### PR TITLE
fix typo in enterprise contract security tab

### PR DESCRIPTION
**Pull request by sahil143**
## Fixes 

https://issues.redhat.com/browse/HAC-3696

## Description

typo fixes

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

image

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Review by simonbaird — CHANGES_REQUESTED** [review]

File: src/components/EnterpriseContractView/SecurityEnterpriseContractTab.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ export const SecurityEnterpriseContractTab: React.FC<{ pipelineRun: string }> =
           Enterprise Contract is a set of tools for verifying the provenance of application
           snapshots and validating them against a clearly defined policy.
           <br />
-          Enterprise Contract policy is defined using the rego policy langauage and is described in{' '}
+          Enterprise Contract policy is defined using the repo policy language and is described in{' '}
```
[the remark this answers is not in the document]
Comment (simonbaird) [reply src/components/EnterpriseContractView/SecurityEnterpriseContractTab.tsx:145]: "Rego" is not a typo, that's what the OPA policy language is called.

[the remark this answers is not in the document]
Comment (simonbaird) [reply src/components/EnterpriseContractView/SecurityEnterpriseContractTab.tsx:145]: I'm not 100% sure about the capitalization, but perhaps "Rego" is better than "rego".

[the remark this answers is not in the document]
Comment (sahil143) [reply src/components/EnterpriseContractView/SecurityEnterpriseContractTab.tsx]: `Rego` is capitalized in the documentation so kept it that way.

**Review by simonbaird — APPROVED** [review]

**Pull request merged**